client/my-sites/domains/domain-management/dataviews/dataviews-fields/domain-field.tsx

```diff
@@ -0,0 +1,83 @@
+import { PartialDomainData } from '@automattic/data-stores';
+import { PrimaryDomainLabel } from '@automattic/domains-table';
+import { domainInfoContext } from '@automattic/domains-table/src/utils/constants';
+import { getDomainTypeText } from '@automattic/domains-table/src/utils/get-domain-type-text';
+import { domainManagementLink as getDomainManagementLink } from '@automattic/domains-table/src/utils/paths';
+import { Button } from '@wordpress/components';
+import { useI18n } from '@wordpress/react-i18n';
+import { navigate } from 'calypso/lib/navigate';
+import { addQueryArgs } from 'calypso/lib/url';
+import { useDomainsDataViewsContext } from '../use-context';
+import useQueryParams from '../use-query-params';
+
+interface Props {
+	domain: PartialDomainData;
+	isAllSitesView: boolean;
+	selectedFeature?: string;
+	openDomainPane?: ( domain: PartialDomainData ) => void;
+}
+
+const DomainField = ( {
+	domain: partialDomain,
+	isAllSitesView,
+	selectedFeature,
+	openDomainPane,
```

Comment:
What's this property? I don't see it defined anywhere ([here](https://github.com/Automattic/wp-calypso/blob/542102d8cda107fd8642044650b271a53b55cca7/client/my-sites/domains/domain-management/list/bulk-all-domains.tsx#L682)).

Reply:
Fixed here: https://github.com/Automattic/wp-calypso/pull/97677/commits/b169ca7454da0519f160c63000652e01ceb94534